ister_gui.py

```diff
@@ -713,7 +714,19 @@ def handler(self, config):
     def build_ui_widgets(self):
         """Build ui handler
         The last urwid Divider helps the tab movement to work"""
+        if os.environ.get('https_proxy'):
+            https_text = os.environ['https_proxy']
+        else:
+            https_text = ''
+
+        if os.environ.get('http_proxy'):
+            http_text = os.environ['http_proxy']
+        else:
+            http_text = ''
+
```

Comment:
I set the environment variables so that subsequent commands (like curl below) can grab them from the environment. I would run into an issue trying to figure out which proxy to pass to curl if I allowed the user to set both.